core/src/main/java/com/linecorp/armeria/server/DefaultRoute.java

```diff
@@ -85,9 +85,7 @@ final class DefaultRoute implements Route {
         if (!headerPredicates.isEmpty()) {
             complexity += 1 << 3;
         }
-        if (!excludedRoutes.isEmpty()) {
-            complexity += 1 << 4;
-        }
+        // Exclude excludedRoutes from the complexity so that it isn't used when finding duplicate route.
```

Comment:
I *think* there should be no issue with this change as the complexity serves as a hint for routing (the behavior doesn't really change)

Let me know if I'm wrong though

Reply:
That is correct. 👍